src/components/SearchBar.tsx

```diff
@@ -31,61 +33,65 @@ type SearchBarCommandsType = {
   cancelSearch: (viewRef: React.ElementRef<typeof NativeSearchBar>) => void;
 };
 
-class SearchBar extends React.Component<SearchBarProps> {
-  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
+const SearchBar = React.forwardRef<SearchBarCommands, SearchBarProps>(
+  function SearchBar(props, ref) {
+    const nativeSearchBarRef = React.useRef<SearchBarCommands | null>(null!);
+    React.useImperativeHandle(ref, () => ({
+      ...nativeSearchBarRef.current,
```

Comment:
Why do we write `...nativeSearchBarRef.current` here? My guess was to expose usual `View` ref methods, however this is not reflected in the reference type so user would get an error any way. Is there different reason?

Reply:
I thought it's a must have for useImperativehandle, but It seems to work fine without it

Reply:
https://github.com/software-mansion/react-native-screens/pull/2132/commits/351a18702deb8873ad48cd9cec3686c29c26741c